src/dotnet/Chat.UI.Blazor/Components/components.css

```diff
@@ -190,3 +190,35 @@ body.hoverable .people-tab .avatar-badge:hover {
     @apply flex-y items-end gap-y-0.5;
     @apply group-hover:hidden;
 }
+
+.author-modal .header-tile-item .tile-item-body-content {
+    @apply max-h-8;
+    @apply text-title-1 text-02;
+}
+.author-modal .tile-item-body-content {
+    @apply text-01;
+}
+.author-modal .first-tile-item {
+    @apply gap-x-4;
+}
+.author-modal .first-tile-item .tile-item-icon {
+    @apply max-w-fit;
+}
+body.hoverable .author-modal .tile,
+body.hoverable .author-modal .tile:hover {
+    @apply p-0;
+    @apply shadow-none;
+}
+body.narrow .author-modal {
+    @apply absolute bottom-6 left-6 right-6;
+    @apply w-auto h-auto;
+    @apply p-0;
+    @apply rounded-lg;
+}
+body.narrow .author-modal .dialog-buttons {
```

Comment:
Looks like vertical padding can be applied generally for `dialog-buttons` in narrow mode, not only inside `author-modal`. May be we can simplify other styles as well. What do you think?